json-schema/sigma-filters-schema.json

```diff
@@ -53,38 +53,12 @@
     },
     "filter": {
       "type": "object",
-      "required": ["rules","selection","condition"],
+      "required": [
+        "rules",
+        "selection",
+        "condition"
+      ],
       "description": "A set of search-identifiers that represent properties of searches on log data",
-      "additionalProperties": {
```

Comment:
This is unnecessary here. The `filter` is an object with required properties, not additional ones.